spec/requests/providers/capital_income_assessment_results_spec.rb

```diff
@@ -278,6 +278,61 @@
         expect { subject }.to raise_error(/Unknown capital_assessment_result/)
       end
     end
+
+    context 'enable_employed_journey is true' do
+      let(:before_tasks) do
+        Setting.setting.update!(enable_employed_journey: true)
+        allow_any_instance_of(Provider).to receive(:employment_permissions?).and_return(true)
+        login_provider
+        subject
+      end
+
+      context 'applicant has employment(s)' do
+        let(:cfe_result) { create :cfe_v4_result, :with_employments }
+        it 'displays the employment income' do
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.other_income.title'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.title'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.monthly_income_before_tax'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.benefits_in_kind'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.tax'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.national_insurance'))
+          expect(unescaped_response_body).to include(I18n.t('providers.capital_income_assessment_results.employment_income.fixed_employment_deduction'))
```

Comment:
This tests that the text is being displayed, which is good, but is it also worth testing that the correct values are displayed too? ie have a test like 
```
expect(unescaped_response_body).to include(cfe_result.employment_income_gross_income)
```
for each of the types of employment income.

It's probably overkill, so this is fine to be released without the extra tests if you don't want to add them.

Reply:
Thanks @mpw5. I agree that we should test that the correct values are displayed. I have added some tests based on your suggestion - are you happy with these?